jenkins/spark-tests.sh

```diff
@@ -66,7 +66,6 @@ BASE_SPARK_SUBMIT_ARGS="$BASE_SPARK_SUBMIT_ARGS \
     --executor-memory 12G \
     --total-executor-cores 6 \
     --conf spark.sql.shuffle.partitions=12 \
-    --conf spark.task.maxFailures=1 \
```

Comment:
in nightly integration tests, we explicitly set env `SPARK_VER` in pipelines for spark artifacts download, we could set this config based on `SPARK_VER` like https://github.com/NVIDIA/spark-rapids/blob/branch-21.08/jenkins/spark-tests.sh#L100

Reply:
I think it's better to not rely on it directly, so we have the logic that works for both: pipelines and local direct invocations of integration_tests/run_pyspark_from_build.sh

Reply:
+1, we will need update https://github.com/NVIDIA/spark-rapids/blob/branch-21.08/jenkins/spark-tests.sh#L97-L103 to use run_pyspark_from_build.sh here

Reply:
@pxLi good catch, do you mind filing an issue because it won't happen in this PR

Reply:
@gerashegalov @pxLi I duplicated some logic in spark-tests.sh. Yes, I tried combining, and it's probably OK, but that seems like a different issue. https://github.com/NVIDIA/spark-rapids/pull/2792/commits/94cca6b524243b6f80e9e7f3eb6c03c03c8ed08e

Reply:
I did base it off SPARK_VER, as it was before in spark-tests.sh.